internal/gosmith/type.go

```diff
@@ -43,7 +43,7 @@ type Type struct {
 	utyp           *Type   // underlying type
 	styp           []*Type // function arguments
 	rtyp           []*Type // function return values
-	elems          []*Var  // struct fileds and interface methods
+	elems          []*Var  // struct fields and interface methods
 	literal        func() string
 	complexLiteral func() string
 
```

Comment:
the todo below mentions `pointerTo` -- I can't tell if that's related to `pointee` above.

My tooling is much happier with `pointerTo` (even Google Chrome is happy w/ it)